### ISSUE TITLE
Notarize plugin to prevent malware flag in macOS Catalina

### ISSUE THREAD

**Issue by drawcard** [webmproject/WebPShop#10]
Came here to report a similar issue to #3 and #9, but wanted to add that it would be wonderful if the plugin could be notarized by Apple, so it can prevent errors like this.

**Comment by y-guyon** [comment]
I agree; it is planned but not considered an emergency as there is a workaround (mentioned in [issue #3](https://github.com/webmproject/WebPShop/issues/3)).

**Comment by drawcard** [comment]
No worries. Thanks for replying!
